docs/node/daemon/config/settings.mdx

```diff
@@ -90,49 +102,53 @@ This section contains configuration specific to the CometBFT-based blockchain.
 
-      - `seed_mode`: A toggle to run in a special "seed mode" where kwild constantly crawls the network to obtain a large address book, which it shares with incoming nodes before disconnecting them. A node running with this setting would be used only to bootstrap other nodes via their `seeds` setting. This is not appropriate for a validator or an RPC service provider.
+      - `seed_mode`: A toggle to run in a special "seed mode" where kwild constantly crawls the network to obtain a large address book, which it shares with incoming nodes before disconnecting them. A node running with this setting would be used only to bootstrap other nodes via their `seeds` setting. This is not appropriate for a validator or an RPC service provider. Default is `false`.
 
       - `addr_book_strict`: A toggle for enforcing strict address routability rules. By default, nodes with routable addresses are considered for connection. However, if this setting is disabled, non-routable IP addresses, like addresses in a private network can be added to the address book.
 
-      - `max_num_inbound_peers`: This variable specifies the maximum number of inbound peers.
+      - `max_num_inbound_peers`: This variable specifies the maximum number of inbound peers. Default is 40.
 
-      - `max_num_outbound_peers`: This variable specifies the maximum number of outbound peers to connect to, excluding persistent peers.
+      - `max_num_outbound_peers`: This variable specifies the maximum number of outbound peers to connect to, excluding persistent peers. Default is 10.
 
-      - `unconditional_peer_ids`: This variable specifies a comma separated list of node ID's that will always have connections (re)established, regardless of the connection limits.
+      - `unconditional_peer_ids`: This variable specifies a comma separated list of node ID's that will always have connections (re)established, regardless of the connection limits. No default.
 
-      - `allow_duplicate_ip`: A toggle to prevent multiple peers from using the same IP address.
+      - `allow_duplicate_ip`: A toggle to prevent multiple peers from using the same IP address. Default is `true`.
 
    ### [chain.mempool]
    
       This section discusses the configurations for the blockchain's transaction mempool.
 
-      - `size`: This variable specifies the mempool's maximum transaction count.
+      - `size`: This variable specifies the mempool's maximum transaction count. Default is 5000.
+
+      - `max_tx_byte`: This variable specifies the maximum size of any one transaction in the mempool. Default is 4MB.
 
-      - `max_txs_bytes`: This variable specifies the limit on the total size of all transactions in the mempool.
+      - `max_txs_bytes`: This variable specifies the limit on the total size of all transactions in the mempool. Default is 512 MB.
```

Comment:
To be pedantic, MiB.  `MaxTxsBytes: 1024 * 1024 * 512, // 512 MiB`  so not 512,000,000, but 536,870,912 or 2^29

Reply:
Thank you.